django_plastic_tickets/migrations/0002_auto_20210126_2216.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.1.2 on 2021-01-26 21:16
```

Comment:
Please merge all migrations. There is now point in keeping them in this change, if you are redefining the model anyway.
There should only be the (new) initial one.